django_downloadview/io.py

```diff
@@ -3,9 +3,11 @@
 from __future__ import absolute_import
 import io
 
+from django.utils.encoding import force_text, force_bytes
 
-class StringIteratorIO(io.TextIOBase):
-    """A dynamically generated StringIO-like object.
+
+class TextIteratorIO(io.TextIOBase):
```

Comment:
TextIteratorIO is more explicit than StringIteratorIO: deals with text (formerly unicode) only.